### PR TITLE
Third party validation payment

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/payments/third-party-validation/

### CODE REVIEW

File: documents/payment.md [hunk 1]
```diff
@@ -630,6 +630,196 @@ Doc reference [doc](https://razorpay.com/docs/payments/payment-methods/cards/aut
...
+  "amount": "100",
+  "currency": "INR",
+  "email": "gaurav.kumar@example.com",
+  "contact": "9123456789",
+  "order_id": "order_JkVtS9XJdk2u0B",
+  "method": "netbanking",
+  "bank": "HDFC"
+}
+
+Razorpay::Payment.create_json_payment(param_attr)
```
Comment (sofiya02) [inline documents/payment.md:646]: throwing error please check with order order_JrxDnTVcxeY8U0

File: documents/payment.md [hunk 2]
```diff
@@ -630,6 +630,196 @@ Doc reference [doc](https://razorpay.com/docs/payments/payment-methods/cards/aut
...
+    "note_key": "value1"
+  },
+  "upi": {
+    "flow": "collect",
+    "vpa": "gauravkumar@exampleupi",
+    "expiry_time": 5
+  }
+}
+
+Razorpay::Payment.create_upi(para_attr)
```
Comment (sofiya02) [inline documents/payment.md:702]: - Error on create_upi 
- Also change variable name from para_attr to param_attr

File: documents/payment.md [hunk 3]
```diff
@@ -630,6 +630,196 @@ Doc reference [doc](https://razorpay.com/docs/payments/payment-methods/cards/aut
...
+    "description": "Test flow",
+    "notes": {
+      "purpose": "UPI test payment"
+    },
+    "upi": {
+      "flow" : "intent"
+    }
+}
+
+Razorpay::Payment.create_upi(para_attr)
```
Comment (sofiya02) [inline documents/payment.md:752]: Error on create_upi
Also change variable name from para_attr to param_attr

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/payment.md (hunk 1 above)
Comment (ankitdas13) [reply documents/payment.md:646]: use your api cred with these order id's
`order_Jrxv1pBFOyWjMC, order_JrxvRq2ovnYDDE ,order_JrxvuFxc5JRkKG`

File: documents/payment.md (hunk 2 above)
Comment (ankitdas13) [reply documents/payment.md:702]: try this code snippet with these order id's
`order_JryDO5A3pU0533, order_JryDcxfWtyWuQO, order_JryDlAaPomSXi0`

para_attr = {
  "amount": 100,
  "currency": "INR",
  "order_id": "order_JryClMTXJvYc3H",
  "email": "gaurav.kumar@example.com",
  "contact": "9123456789",
  "method": "upi",
  "customer_id": "cust_JkRs374KyewdSU",
  "save": 1,
  "ip": "192.168.0.103",
  "referer": "http",
  "user_agent": "Mozilla/5.0",
  "description": "Test flow",
  "notes": {
    "note_key": "value1"
  },
  "upi": {
    "flow": "collect",
    "vpa": "success@razorpay",
    "expiry_time": 5
  }
}

Razorpay::Payment.create_upi(para_attr)

File: documents/payment.md (hunk 3 above)
Comment (ankitdas13) [reply documents/payment.md:752]: try with these order id's . i will share api cred for this

`order_JryIs7T7t7zeWV, order_JryJAT9qDE6dqZ, order_JryJHXlh4WELs1`

para_attr = {
    "amount": 100,
    "currency": "INR",
    "order_id": "order_JryExkN1YXfugO",
    "email": "gaurav.kumar@example.com",
    "contact": "9090909090",
    "method": "upi",
    "ip": "192.168.0.103",
    "referer": "http",
    "user_agent": "Mozilla/5.0",
    "description": "Test flow",
    "notes": {
      "purpose": "UPI test payment"
    },
    "upi": {
      "flow": "intent"
    }
}

Razorpay::Payment.create_upi(para_attr)

**Pull request merged**